crate/cli/Cargo.toml

```diff
@@ -24,6 +24,7 @@ cosmian_kms_client = { path = "../client" }
 cosmian_kms_utils = { path = "../utils" }
 cosmian_logger = { path = "../logger" }
 der = { workspace = true }
+hex = "0.4"
```

Comment:
is hex encoding is really required? Can input salt be encoded in base64 instead?

Reply:
Yes sure, commit f152a523ae5f3070783edd03674145a6b82b9f44 should fix this.